### PR TITLE
#172499577 Users should be able to switch between multiple languages

### PR DESCRIPTION
**Pull request by blaise82**
#### What does this PR do?
Allow user to switch between two languages 

#### Description of Task to be completed?
- set up locale 
- allow user to change language on click

#### How should this be manually tested?
- Close this repo
- Run npm install 
- Run npm run dev to start the development server 

#### What are the relevant pivotal tracker stories?
[172499577](https://www.pivotaltracker.com/story/show/172499577)
#### Screenshots (if appropriate)

screencapture-localhost-8080-2020-04-26-13_06_47
Screen Shot 2020-04-26 at 18 33 30

### CODE REVIEW

**Review by AbdoulNuru — APPROVED** [review]

**Review by AbdoulNuru — APPROVED** [review]

File: src/languages/messages/en.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "bn-value": "Making company travel and accommodation easy and convenient.",
+    "sign-in": "Sign In",
+    "fill-form": "Please fill the form",
+    "no-account": "Have no account?",
+    "forget-password": "Forgot password?",
+    "or": "Or",
+    "password": "Password"
+}
```
Comment (AJAkimana) [inline src/languages/messages/en.json]: Add new line

File: src/languages/messages/fr.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "bn-value": "Rendre le voyage et l'hébergement de l'entreprise faciles et pratiques.",
+    "sign-in": "Se connecter",
+    "fill-form": "Veuillez remplir le formulaire",
+    "no-account": "Vous n'avez pas de compte?",
+    "forget-password": "Mot de passe oublié?",
+    "or": "Ou",
+    "password": "Mot de passe"
+}
```
Comment (AJAkimana) [inline src/languages/messages/fr.json]: Add new line

File: src/styles/language.styles.module.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.langue_button{
+    margin-right: 0.7em;
+    float: left;
+
+}
+.container{
+    margin-top: 1em;
+    float: right;
+    margin-right: 1em;
+}
```
Comment (AJAkimana) [inline src/styles/language.styles.module.css]: Add new line

**Review by AJAkimana — COMMENTED** [review]
Add new lines on some files

**Pull request merged**